packages/debugger/style/variables.css

```
@@ -114,6 +114,15 @@
   color: var(--jp-content-font-color0);
 }
 
+.jp-VariableRendererPanel {
+  overflow: auto;
+}
+
+.jp-VariableRendererPanel-renderer {
+  overflow: auto;
+  height: 100%;
```

@firai just highlighting the suggestion which you might have missed from https://github.com/jupyterlab/jupyterlab/pull/12968#discussion_r950181192. I can confirm that `overflow: visible` works well on Chromium too, so I think we can go that way for simplicity.

```suggestion
  overflow: visible;
```

---

Regardless of whether you prefer overflow auto + height 100% or overflow visible, please update the top-level comment to reflect the changes applied after review before we merge :)

---

Thanks for the suggestion, @fcollonval, and thanks for pointing this out, @krassowski. I did miss the additional comment. `auto` + `100%` height might be easier to understand in the future; this particular `visible` behavior seems non-obvious? What do you all think?

Top level info has been updated, though the screenshot is from a local test build. Binder has been giving me server-spawning errors in the last few hours for some reason.